### PR TITLE
Added map Limited Access

### PR DESCRIPTION
**Pull request by Mailaender**
http://resource.openra.net/maps/22843/

### CODE REVIEW

**Review by reaperrr — DISMISSED** [review]

**Comment by pchote** [comment]
One nit that blocks merging:
* Please remove the bridge huts from the elevated bridge (it isn't repairable, or even really a bridge).

One wish that doesn't have to block merging, but would be a good idea:
* Please add a case to the map importer to replace template `717` (dcliff01) with `403` and `404` (ramp01/ramp02), and `718` with `405` and `406`.  Otherwise we will need to edit the map in FinalSun to remove the destroyable cliffs and then reimport it from scratch.

**Comment by pchote** [comment]
IMO the destroyable cliffs on this map are a problem.  I won't insist on fixing the importer in this PR, but it should be fixed in *some* PR before we merge this.  Otherwise we will need to edit the map in FinalSun and then reimport it from scratch, effectively wasting this one.

Template `717` (dcliff01) should be replaced with `403` and `404` (ramp01/ramp02), and `718` with `405` and `406`.  The indices won't match up, so we will need to define a translation table for those too.

File: mods/ts/maps/access/map.yaml
Comment (pchote) [inline mods/ts/maps/access/map.yaml]: This is one cell too long.

**Comment by Mailaender** [comment]
Yep, this map relies a lot on #13829.

**Comment by pchote** [comment]
#14046

**Comment by reaperrr** [comment]
@Mailaender: Since #14046 is closed and I doubt anyone else than @pchote will be able to come up with a satisfying solution, I think this PR should be closed for now. After some thinking, I'd say merging the map with indestructible cliffs feels wrong since they seem relatively important on this map, while reimporting it now with normal ramps (using #14046) would require another reimport and another review later.

I suggest keeping it around locally or here so the bridges won't have to be redone when reimporting later, but I don't think there's a point in keeping the PR open and updating it until we have a solid solution for destructible cliffs.

**Pull request closed without merging**
